rules/windows/execution_script_executing_powershell.toml

```diff
@@ -30,9 +30,13 @@ event.category:process and event.type:(start or process_started) and
 [[rule.threat]]
 framework = "MITRE ATT&CK"
 [[rule.threat.technique]]
-id = "T1193"
+id = "T1566"
+name = "Phishing"
+reference = "https://attack.mitre.org/techniques/T1566/"
+[[rule.threat.technique.subtechnique]]
+id = "T1566.001"
 name = "Spearphishing Attachment"
-reference = "https://attack.mitre.org/techniques/T1193/"
+reference = "https://attack.mitre.org/techniques/T1566/001/"
 
 
 [rule.threat.tactic]
```

Comment:
should be initial access

Reply:
I have seen this issue before. I think it is a GH glitch or something: this file is updated already with `Initial Access` but it is not reflecting here. If you go to the source file, you can see it is correct: https://github.com/brokensound77/detection-rules/blob/update-rule-threat-mappings/rules/windows/initial_access_script_executing_powershell.toml

This is happening to several updates from dc50a174dd679e2e70387202464c74699e8723fe

Any ideas @rw-access ?

Reply:
could it be related to the file name change?

Reply:
so it looks like the comment is on L42 on the original file. looks good on the GH diff now. i think because the comment wasn't on a line that actually changed, you don't see the "Outdated" that you usually would.

i'll resolve this convo